pymongo/ocsp_support.py

```diff
@@ -132,6 +148,10 @@ def _verify_signature(
             key.verify(signature, data, algorithm)  # type: ignore[arg-type]
         elif isinstance(key, _EllipticCurvePublicKey):
             key.verify(signature, data, _ECDSA(algorithm))  # type: ignore[arg-type]
+        elif isinstance(key, _X25519PublicKey) or isinstance(
+            key, _X448PublicKey
+        ):  # Curve25519 and Curve448 keys do not require verification
```

Comment:
Neat! Can you give a reference for this?

Reply:
For Curve448: https://crypto.stackexchange.com/questions/91710/how-do-i-validate-curve448-public-keys

For Curve25519: https://crypto.stackexchange.com/questions/87709/curve25519-key-validation and https://cr.yp.to/ecdh.html